crates/interledger-service/src/lib.rs

```diff
@@ -52,6 +52,9 @@ pub trait Account: Clone + Send + Sized + Debug {
 
     fn id(&self) -> Self::AccountId;
     fn username(&self) -> &Username;
+    fn client_address(&self) -> &Address;
```

Comment:
If we're changing this already, I really want to rename this method. `client_address` is what it's called in ILDCP but this goes beyond ILDCP now

```suggestion
    fn ilp_address(&self) -> &Address;
```